src/components/Contexts/StepperProvider.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useState } from 'react'
+
+import useCycle from 'cozy-ui/transpiled/react/hooks/useCycle'
+
+const StepperContext = createContext()
+
+const StepperProvider = ({ children }) => {
+  const [maxStep, setMaxStep] = useState(1)
+
+  // REVIEW Check relevance to use `useCycle`
+  const [currentStep, setPrev, setNext] = useCycle(1, 1, maxStep)
```

Comment:
I agree, as you already check limits in `previousStep` and `nextStep`, useCycle is not needed here.